ui/src/components/Form/TextField/style.module.scss

```diff
@@ -3,12 +3,35 @@
 .input-field-container {
   flex-grow: 1;
   position: relative;
+  width: 100%;
+
+  input {
+    @extend .input;
+  }
+}
+
+.login-form {
+  flex-grow: 1;
+  position: relative;
+  width: 100%;
 
   input {
     @extend .input;
   }
 }
 
+.org-add-form {
```

Comment:
This looks like an organization-form specific name, in generic `TextField` component. We shouldn't add use-case specific styles in generic components. I'd be better if these styles are overridden by parent, and kept in the parent itself.